src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -302,60 +346,138 @@ val qualityFormat: Option[FastqQualityFormat] = None,
     // Write the records out in its own thread
     iterator.foreach { demuxRecord =>
       demuxRecord.sampleInfo.metric.increment(numMismatches=demuxRecord.numMismatches)
-      val writer = demuxRecord.sampleInfo.writer
+      val writer = sampleToWriter(demuxRecord.sampleInfo.sample)
       demuxRecord.records.foreach { rec =>
-        writer.addAlignment(rec)
-        progress.record(rec)
+
+        val readQualsBytes = rec.quals.getBytes
+        convertQuality(readQualsBytes, qualityFormat, rec.name)
```

Comment:
Maybe we should note in the usage that even if you are outputting fastq, your quality scores will be standardized to sanger/SAM format?